src/test/ui/consts/min_const_fn/min_const_fn_fn_ptr.rs

```diff
@@ -19,6 +19,10 @@ struct Hide(HasPtr);
 fn field() {}
 
 const fn no_inner_dyn_trait(_x: Hide) {}
+const fn no_inner_dyn_trait2(x: Hide) {
+    x.0.field;
+//~^ ERROR function pointers in const fn
+}
 const fn no_inner_dyn_trait_ret() -> Hide { Hide(HasPtr { field }) }
 //~^ ERROR function pointers in const fn
 
```

Comment:
So you already have the erroring tests:

```rust
const fn no_dyn_trait(_x: &dyn std::fmt::Debug) {}
const fn no_fn_ptrs(_x: fn()) {}
```

This is good, because it leaves open the possibility that this means &dyn const std::fmt::Debug or const fn(); If we didn't reject this, then users could pass in values where you don't have a const implementation of std::fmt::Debug, and so we wouldn't be able to change this in the future. However, as for situations where you have:

```rust
struct Hide<'a>(&'a dyn Debug);
const fn foo(_x: Hide) {}
```

... this would only prevent a situation in the future where we decided to split up our nominal type universe into one for the const restriction and one for impure... This seems like a very unlikely future... as such... I think the tests you have currently are sufficient.

So you don't need to implement it.